snippets/microshift-unsupported-config-warn.adoc

```diff
@@ -0,0 +1,15 @@
+// Text snippet included in the following modules:
+//
+// * modules/microshift-updating-rpms-y.adoc
+// * modules/microshift-embed-ostree-enable-eus-repos.adoc
+// * assemblies/microshift-update-options.adoc
+// * assemblies/microshift-embed-in-rpm-ostree.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[WARNING]
+====
+Keeping component versions in a supported configuration of {op-system-bundle} can require updating {microshift-short} and {op-system-base} at the same time. Ensure that your version of {op-system-base} is compatible with the version of {microshift-short} you are updating to, especially if you are updating {microshift-short} across two minor versions. Otherwise, you can create an unsupported configuration, break your cluster, or both. For more information, see the link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/installing/microshift-install-rpm#install-rpm-rhde-compatibility-table[Red Hat Device Edge release compatibility matrix].
```

Comment:
It looks like you have the correct link here, but when I clicked it in the preview, it took me to the top of the page that the matrix is on, instead of taking me directly to the matrix.